guacml/models/xgboost.py

```diff
@@ -1,25 +1,25 @@
 # ToDo: Deprecation warning because xgboost import cross_validation
 import xgboost as xgb
 import numpy as np
+import pandas as pd
 
 from guacml.enums import ProblemType
 from guacml.models.base_model import BaseModel
-from guacml.models.hyper_param_info import HyperParameterInfo
 from guacml.preprocessing.column_analyzer import ColType
 from hyperopt import hp
 
 
-
 class XgBoost(BaseModel):
+
```

Comment:
Was the extra line here intentional? I noticed you don't usually put one after a class declaration.